File: higher-or-lower/higherOrLower.js
const higherOrLower = (string) => {
    stringArr = string.split(' ');

    return Math.max(... stringArr) + " " + Math.min(... stringArr);
};

module.exports = { higherOrLower };

//https://www.codewars.com/kata/554b4ac871d6813a03000035/train/javascript

File: python-venv/higher_or_lower/kata_function/higher_or_lower.py
def higher_or_lower(string):
    stringArr = string.split()
    numbers = list(map(lambda x: int(x), stringArr))

    return str(max(numbers)) + " " + str(min(numbers))


#https://www.codewars.com/kata/554b4ac871d6813a03000035/train/python